lib/wallet/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -382,10 +383,8 @@ constructUnsignedTx
     -> (Maybe Cardano.TxMetadata, [Cardano.Certificate])
     -> (Maybe SlotNo, SlotNo)
     -- ^ Slot at which the transaction will optionally start and expire.
-    -> RewardAccount
-    -- ^ Reward account
-    -> Coin
-    -- ^ An optional withdrawal amount, can be zero
+    -> Withdrawal
+    -- ^ Withdrawal
```

Comment:
This comment doesn't add any new information that isn't already contained in the type.

Reply:
It bugs me when all arguments have comment but one though